forms-flow-web/src/components/Form/List.js

```diff
@@ -428,12 +428,26 @@ const List = React.memo((props) => {
     }
   };
 
+
   const fileUploaded = async (evt) => {
     FileService.uploadFile(evt, async (fileContent) => {
-      if (fileContent) {
-        dispatch(setFormUploadList(fileContent?.forms || []));
+      if ("forms" in fileContent) {
+        var formToUpload = fileContent;
+      }
+      else {
```

Comment:
Can reduce this to a single statement say
 let obj = {a: 1, b: 2, c: 3, d: 4, e: 5 };

['c', 'e'].forEach(e => delete obj[e]);

// obj is now {a:1, b:2, d:4}

Reply:
And try avoiding var . Use let or const more in blocks

Reply:
sure